pkg/xds/generator/util.go

```diff
@@ -0,0 +1,13 @@
+package generator
```

Comment:
how about `resource_names.go`. For me it would be more clear what code it contains.

Reply:
renamed